src/app/build.gradle

```diff
@@ -1,22 +1,7 @@
-import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
```

Comment:
This is totally wrong syntax, not sure how this ended up in here, my bad!

Reply:
I did wonder about that!